chunky/src/java/se/llbit/math/Octree.java

```diff
@@ -195,27 +207,14 @@ public DataNode(int type, int data) {
     }
   }
 
-  /**
-   * Recursive depth of the octree
-   */
-  public final int depth;
-
-  /**
-   * Root node
-   */
-  public final Node root;
-
   /**
    * Timestamp of last serialization.
    */
   private long timestamp = 0;
 
-  private final Node[] parents;
-  private final Node[] cache;
-  private int cx = 0;
-  private int cy = 0;
-  private int cz = 0;
-  private int cacheLevel;
+  private OctreeImplementation implementation;
+
+  static private boolean usePacked = true;
```

Comment:
Maybe we should make this configurable via a Java system variable (`usePacked = !System.getProperty("chunky.useLegacyOctree").equals("true")`) so that users can change this if they encounter issues by launching Chunky with `-Dchunky.useLegacyOctree=true`

Reply:
I'm not familiar with that but that seems like a good idea

Reply:
Implemented and works fine